neo/io/proxyobjects.py

```diff
@@ -12,6 +12,7 @@
 
 import numpy as np
 import quantities as pq
+import loggging
```

Comment:
too many 'gggg's (-:

Reply:
that, of course, explains the failing tests ^^